### PR TITLE
Update weekly and nightly tests flags

### PR DESCRIPTION
**Pull request by vlad-nazarov**

### CODE REVIEW

File: dev/bazel/dal.bzl [hunk 1]
```diff
@@ -344,22 +343,13 @@ def _test_eternal_datasets_args(framework):
...
-            ],
-        })
-    return []
-
-def _test_weekly_args(framework):
+def _test_filter_args(framework):
     if framework == "catch2":
         return _select({
+            "@config//:test_nightly_enabled": ["~[weekly]"],
             "@config//:test_weekly_enabled": [],
```
Comment (rlnx) [inline dev/bazel/dal.bzl:350]: What is behavior if both options `--test_nightly` and `--test_weekly` are passed?

Comment (vlad-nazarov) [reply dev/bazel/dal.bzl:350]: The tests will not run with the error that only one of these flags is needed.

Comment (rlnx) [reply dev/bazel/dal.bzl:350]: I've tried this, error message is not very clear, but probably fine for now

**Review by rlnx — APPROVED** [review]

**Pull request merged**